examples/basic/variables.tf

```diff
@@ -1,33 +1,35 @@
-########################################################################################################################
-# Input variables
-########################################################################################################################
+####################
+# Cluster variables
+####################
+variable "cluster_id" {
+  type        = string
+  description = "The name or ID of the cluster to be used"
+  default     = "my-cluster-name"
+}
 
 variable "ibmcloud_api_key" {
   type        = string
-  description = "The IBM Cloud API Key"
+  description = "IBM Cloud API key"
   sensitive   = true
 }
 
-variable "region" {
+variable "ibmcloud_region" {
   type        = string
-  description = "Region to provision all resources created by this example"
+  description = "IBM Cloud region where all resources will be deployed"
   default     = "us-south"
 }
 
-variable "prefix" {
```

Comment:
I suggest to add pack `prefix` here so the tests can use a unique prefix for every run to avoid name clashing